ui/src/app/pages/profile/demographic-survey-panel.tsx

```diff
@@ -13,7 +13,7 @@ export const DemographicSurveyPanel = (props: Props) => {
   const { demographicSurveyCompletionTime } = props;
 
   return (
-    <div style={styles.panel}>
+    <div style={styles.panel} data-test-id='demographic-survey'>
```

Comment:
we should not be adding new data-test-ids.  Please write the test in react-testing-library so it does not require this.